### PR TITLE
Add par_regions_bed input to DeepVariant

### PR DESCRIPTION
**Pull request by fellen31**
Forgot to add this while updating DeepVariant to 1.6.1. 

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Review by SPPearce — APPROVED** [review]

File: modules/nf-core/deepvariant/main.nf [hunk 1]
```diff
@@ -12,6 +12,7 @@ process DEEPVARIANT {
     tuple val(meta2), path(fasta)
     tuple val(meta3), path(fai)
     tuple val(meta4), path(gzi)
+    tuple val(meta5), path(par_bed)
```
Comment (FriederikeHanssen) [inline modules/nf-core/deepvariant/main.nf:15]: can you update the meta.yml with the new param

**Review by FriederikeHanssen — CHANGES_REQUESTED** [review]

File: modules/nf-core/deepvariant/main.nf (hunk 1 above)
Comment (SPPearce) [reply modules/nf-core/deepvariant/main.nf:15]: Oh, good spot

Comment (fellen31) [reply modules/nf-core/deepvariant/main.nf:15]: Thanks, fixed!

**Comment by fellen31** [comment]
Thanks @SPPearce

**Pull request merged**